### ISSUE TITLE
Add something similar to make checkstyle that zfs has.

### ISSUE THREAD

**Issue by MigeljanImeri** [MigeljanImeri/xdd#41]
Would be nice to have something that we could use to automatically check formatting of code.

**Comment by bwatkinson** [comment]
Yeah that would be nice. It doesn't have to be super complicated as well. We could even start with something like `clang-format`.
